PhysicsTools/PatUtils/interface/SmearedJetProducerT.h

```diff
@@ -247,7 +247,7 @@ class SmearedJetProducerT : public edm::stream::EDProducer<> {
                     }
 
                     std::normal_distribution<> d(0, sigma);
-                    smearFactor = 1. + d(m_random_generator) / jet.energy();
+                    smearFactor = 1. + d(m_random_generator) / jet.pt();
```

Comment:
Sure, I'll do that. However, I'm pretty sure that a jet with `pt() == 0` will crash the system in many many places, but at least, let's keep this code clean!

I also have another fix incoming for the smearing, sorry about that...